modules/plugin/src/main/resources/logback.xml

```diff
@@ -6,18 +6,15 @@
         </encoder>
     </appender>
 
-    <appender name="PLAIN_FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <appender name="PLAIN_FILE" class="ch.qos.logback.core.FileAppender">
         <file>logs/plugin.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <!-- daily rollover -->
+        <!-- <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
```

Comment:
as the `RollingFileAppender` is not used anymore, there is no need to keep the settings for it

Reply:
Removed now 👍🏾